api/src/test/java/com/exelaration/abstractmemery/controllers/MemeControllerTest.java

```diff
@@ -65,4 +68,21 @@ public void MemeControllerTest_WhenNullMemePosts_ExpectStatus400() throws Except
                 .content(new ObjectMapper().writeValueAsString(testMeme)))
         .andExpect(badRequest);
   }
+
+  @Test
+  public void MemeControllerTest_WhenGetMemes_ExpectStatus200andJSONreturn() throws Exception {
```

Comment:
better name would be `getMemesForGallery_WhenMemesExist_ExpectStatus200andJSONreturned()` so that it matches the method name

Reply:
👍